changelog.d/239.change.rst

```diff
@@ -0,0 +1,3 @@
+Added ``type`` argument to ``attr.attr()`` and corresponding ``type`` attribute to ``attr.Attribute``.
+This value can be inspected by third-party tools for type checking and serialization.
```

Comment:
I think this should be more general – we’ll probably use it ourselves too, right?  Maybe just scratch ` by third-party tools` altogether.

OTOH we should stress that nothing happens just by setting it.

Reply:
How about this:

This change paves the way for automatic type checking and serialization features (though as of this release there are no built-in features that make use of it).

Reply:
In changelogs should the project name be monospace?  attrs or `attrs`?

Reply:
Always monospaced `attrs` everywhere when you talk about the project.

Well let's not overthink it, we don't know whether or not we'll make use of it but I can still edit the fragment before the release.